src/pages/ResetPassword.tsx

```diff
@@ -120,6 +124,7 @@ export const ResetPassword = () => {
             onChange={(e) => setOrganizationName(e.target.value)}
             value={organizationName}
             type="text"
+            disabled={SINGLE_TENANT_MODE}
```

Comment:
Rather than `disabled`, we should probably change its visibility to hidden instead.

Reply:
@marcelosalloum You meant that the field shouldn't be rendered, right? If we just change the visibility, there will be a gap in the markup. We need to render `null` instead.

```typescript
{ SINGLE_TENANT_MODE ? null : <Input ... /> }
```